### ISSUE TITLE
Failed to run cargo with exit code 1

### ISSUE THREAD

**Issue by Eh2406** [AtomLinter/linter-rust#96]
Whenever I save I get: 

```
Failed to run cargo with exit code 1
with args:
 rustc -Zno-trans --color never -j 4 --manifest-path C:\Users\...\Cargo.toml --message-format json
See console for more information
```

My console looks like:

```
stdout:
linter-rust.coffee:116 
linter-rust.coffee:117 stderr:
linter-rust.coffee:118 error: Unknown flag: '-Z'

Usage:
    cargo rustc [options] [--] [<opts>...]
```
In cmd:
```
C:\Users\...\src>rustc -V
rustc 1.15.0-nightly (8f02c429a 2016-12-15)
```

**Comment by White-Oak** [comment]
@Eh2406 thanks for the issue!

Try to run cargo check on your project.

**Comment by Eh2406** [comment]
```
>cargo check
   Compiling quadrature v0.1.1 (file:///C:/Users/.../)
    Finished debug [unoptimized + debuginfo] target(s) in 0.57 secs
```
save file in atom. Same error message.

**Comment by White-Oak** [comment]
@Eh2406 thanks! And one more - I assume you are using Windows? If yes, that might be related to  #95. I'll check it out

**Comment by Eh2406** [comment]
oops... ment to mention that. Windows 10 all up to date. I looked at #95 befor reporting, but it has a different error message, so I thought I should report this separately.

**Comment by White-Oak** [comment]
The problem is in linter option `rustc`. It seems that flag `-Z` finally got removed from `cargo rustc`. It is still possible to use that by providing it after all arguments like this: `-- -Zno-trans`. This was used to have simmilar to `cargo-check` behaviour of not building the target.

**Comment by Eh2406** [comment]
So I switched from `cargo command` `rustc` to `build` and that fixed the problem. Thanks!

What is the pan for fixing this in general. Should it select the command based on the rust version?
